src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -1,15 +1,79 @@
 ---
-title: Alerts and applied intelligence transition guide 2022
+title: Alerts and applied intelligence transition guide
 tags:
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - EOL
   - Transition guide
 metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
 
```

Comment:
I wonder if the 2nd paragraph and 2 bullets after "What's changing" should move up to here (after the intro paragraph) so it's easier to see? That particular Explorers Hub post is currently the main one about alerts/AI changes, so I think it could easily apply to both H2 sections.

Reply:
Excellent idea!